### ISSUE TITLE
Override `serviceName` with Docker Swarm Mode

### ISSUE THREAD

**Issue by vovimayhem** [vfarcic/docker-flow-proxy#84]
Hi! I've just gave `docker-flow-proxy` a try. Found the following opportunity:

I'm deploying a stack of services named "blue" to a Swarm using Docker v1.13 `docker stack deploy` command:

```
docker stack deploy --compose-file=docker-compose.yml blue
```

The Compose file contents:
```
version: '3'
networks:
  proxy:
    external: true
services:
  web:
    image: vovimayhem/test-website:blue
    networks:
      - default         # This stack's network
      - proxy           # The App Routing Network (Docker Flow: Proxy)
    deploy:
      labels:
        com.df.notify: "true"
        com.df.distribute: "true"
        com.df.servicePath: /
        com.df.serviceName: blue
        com.df.serviceDomain: example.com
        com.df.port: 80
      placement:
        constraints:
          - node.labels.app-host == true
```

As you may know, `docker stack deploy` will prepend the stack name to each of the services in the stack, which will result in a service called `blue_web` in this case. What I'm trying to do is to override the "subdomain" name that will route requests to the service I created - in this case, to route `http://blue.example.com` to the `blue_web` service.

However this does not work. I did notice in the README's [Reconfigure](https://github.com/vfarcic/docker-flow-proxy#reconfigure) section that `serviceName` is expected to "match the name of the Swarm service". Further more, I tried setting a service alias for the `proxy` network:

```
# (...)
services:
  web:
    # (...)
    networks:
      default:
      proxy:
        aliases:
          - blue
    # (...)
```
...but this didn't work either.

Is there currently a way to achieve this??? Or is it a feature that is currently not present, and would be implemented eventually?

**Comment by vfarcic** [comment]
I'm still brainstorming the best way to support the stack from the proxy. It'll be added soon.

In the meantime, can you change:
`com.df.serviceName: blue`
to
`com.df.serviceName: web_blue`

That should make it work.

**Comment by raarts** [comment]
@vovimayhem, the alias worked for me:

# (...)
services:
  web:
    # (...)
    networks:
      default:
      proxy:
        aliases:
          - blue_web
    # (...)

(where blue is the name of the stack). I think you got the name wrong

**Comment by vfarcic** [comment]
Both solutions should work. In the meantime, I'm waiting for the final 1.13 release before I start updating the projects/books. It'll be a big one :)

**Comment by eemapel** [comment]
I had the similar issue with release candidate version, but with released 1.13 environment variables pointing to other services didn't need hardcoded stack name anymore in compose file.
